scripts/lib/upgrade-zulip-stage-2

```diff
@@ -491,6 +513,8 @@ else:
 
     logging.info("Upgrade complete!")
 
+    run_hooks("post-deploy")
```

Comment:
I'm not entirely sure we want this before the `upgrade complete` and not after, but it's easy to change and I'll have a more informed opinion after trying it.